modulemd/include/private/modulemd-yaml.h

```diff
@@ -867,6 +1375,23 @@ skip_unknown_yaml (yaml_parser_t *parser, GError **error);
     }                                                                         \
   while (0)
 
+/**
+ * EMIT_ARRAY_VALUES:
+ * @emitter: (inout): A libyaml emitter object that is positioned at the start
+ * of where a new sequence will be written.
+ * @error: (out): A #GError that will return the reason for any error.
+ * @key: (in): The name to be used as the identifier for the output sequence.
```

Comment:
Indicate that this function outputs both a key and an array value for that key. So it must be run from within a YAML mapping.

Reply:
There are several other places that need to note this as well.